app/services/stats/management_information_generator.rb

```diff
@@ -21,21 +21,42 @@ def call
 
     attr_reader :format
 
+    def log_error(error)
+      LogStuff.send(:error, 'MI Report generation error has occured:')
```

Comment:
This will create three error log entries for one error. I'd recommend having one error (or debug) entry that has attributes for the details you want:
e.g. 
```
LogStuff.send(
  :error, 
  self.class.name, 
  error_message: "#{error.class} - #{error.message}",
  error_backtrace: error.backtrace.inspect.to_s
)
```

Reply:
👍